### ISSUE TITLE
pubkey of 'solana-keygen new' does not match output of 'solana-keygen pubkey'

### ISSUE THREAD

**Issue by ulrichstark** [solana-labs/solana#17325]
#### Problem
When i run 'solana-keygen new --no-outfile' with no extra passphrase, it generates the keypair like normal and displays a different pubkey compared to the output of 'solana-keygen pubkey prompt://' (and inputting the newly generated seed phrase).

I am using 1.6.9.

I also tested on 1.6.7 with the same command to generate and 'solana-keygen pubkey ask://' to verify the public key.
And they match! But not on 1.6.9 anymore...

**Comment by CriesofCarrots** [comment]
Correct. As of v1.6.9, `solana-keygen pubkey prompt:` returns the solana bip44 base address for the seed phrase, ie. derivation path `m/44'/501'`. You can still access the raw ed25519 keypair using the `ASK` keyword, eg: `solana-keygen pubkey ASK`

cc @t-nelson , let's consider updating `solana-keygen new --no-outfile` to display the the bip44 base address; perhaps not until we update the keypair file syntax.

**Comment by t-nelson** [comment]
> cc @t-nelson , let's consider updating `solana-keygen new --no-outfile` to display the the bip44 base address; perhaps not until we update the keypair file syntax.

Yeah I think updating it with the new keypair file changes makes sense.  I'm hoping to get to those this week.

Perhaps a docs update to mention the legacy, raw usage is sufficient in the mean time?

**Comment by nileshtrivedi** [comment]
I ran into this issue 6 months after it was first reported: https://twitter.com/nileshtrivedi/status/1459422751979151363

**Comment by t-nelson** [comment]
> I ran into this issue 6 months after it was first reported: https://twitter.com/nileshtrivedi/status/1459422751979151363

Use `ASK` not `prompt:`.  `solana-keygen new` doesn't support bip32 yet

**Comment by Mamu9** [comment]
Yes. That's a big trouble for me as I was trying to import private key from phantom and it was saying incorrect format. I was just doing an NFT experiment and uploading in the devnet.. YES solana-keygen new doesn't support bip32 yet. I copied the devnet.jason content and it was all 213,45,23,80.....format  I raised a ticket.

**Comment by xKluil** [comment]
But how can I get private key for pubkey from `solana-keygen new --no-outfile`?

**Comment by CryptoNinja0331** [comment]
thanks.
